src/main/java/org/terasology/navgraph/WalkableBlockFinder.java

```diff
@@ -72,6 +79,16 @@ private void findNeighbors(NavGraphChunk map) {
         }
     }
 
+    /**
+     * Connects Cells to adjacent cells if possible
```

Comment:
Could be worded better for more clarity - something like "Connects [each cell? a cell?] to adjacent cells if possible."